<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NuevaVentana</class>
 <widget class="QDialog" name="NuevaVentana">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imagenes.qrc">
    <normaloff>:/Imagenes/Icono.jpg</normaloff>:/Imagenes/Icono.jpg</iconset>
  </property>
  <widget class="QLabel" name="ldesde">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Desde</string>
   </property>
  </widget>
  <widget class="QLabel" name="lhasta">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Hasta</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeInicio">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumTime">
    <time>
     <hour>22</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
   <property name="minimumTime">
    <time>
     <hour>8</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QLabel" name="lAula">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;B&gt;Aula:&lt;/B&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="boxAulas">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeFin">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>22</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2000</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="maximumTime">
    <time>
     <hour>22</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
   <property name="minimumTime">
    <time>
     <hour>8</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QLabel" name="lDia">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Dia:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="boxDIa">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lcargaHora">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">&lt;B&gt;&lt;/B&gt;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Horas Semanales:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="cargaHoraria">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="boxAnio">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lAnio">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;AÃ±o:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lModal">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Modalidad:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="boxModal">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lLunes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;b&gt;Lunes&lt;b&gt;</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeFin_martes">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>22</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2000</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="maximumTime">
    <time>
     <hour>22</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
   <property name="minimumTime">
    <time>
     <hour>8</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QLabel" name="lhasta_martes">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Hasta</string>
   </property>
  </widget>
  <widget class="QLabel" name="lMartes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Martes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeInicio_martes">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumTime">
    <time>
     <hour>22</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
   <property name="minimumTime">
    <time>
     <hour>8</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QLabel" name="ldesde_martes">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Desde</string>
   </property>
  </widget>
  <widget class="QLabel" name="ldesde_miercoles">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Desde</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeInicio_miercoles">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumTime">
    <time>
     <hour>22</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
   <property name="minimumTime">
    <time>
     <hour>8</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeFin_miercoles">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>22</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2000</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="maximumTime">
    <time>
     <hour>22</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
   <property name="minimumTime">
    <time>
     <hour>8</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QLabel" name="lMiercoles">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Miercoles&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lhasta_miercoles">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Hasta</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeInicio_jueves">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumTime">
    <time>
     <hour>22</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
   <property name="minimumTime">
    <time>
     <hour>8</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeFin_jueves">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>22</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2000</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="maximumTime">
    <time>
     <hour>22</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
   <property name="minimumTime">
    <time>
     <hour>8</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QLabel" name="lJueves">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Jueves&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="ldesde_jueves">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Desde</string>
   </property>
  </widget>
  <widget class="QLabel" name="lhasta_jueves">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Hasta</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeInicio_viernes">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumTime">
    <time>
     <hour>22</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
   <property name="minimumTime">
    <time>
     <hour>8</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeFin_viernes">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>22</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2000</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="maximumTime">
    <time>
     <hour>22</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
   <property name="minimumTime">
    <time>
     <hour>8</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QLabel" name="lViernes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Viernes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="ldesde_viernes">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Desde</string>
   </property>
  </widget>
  <widget class="QLabel" name="lhasta_viernes">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Hasta</string>
   </property>
  </widget>
  <widget class="QLabel" name="ldesde_sabado">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Desde</string>
   </property>
  </widget>
  <widget class="QLabel" name="lSabado">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;SÃ¡bado&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeInicio_sabado">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumTime">
    <time>
     <hour>22</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
   <property name="minimumTime">
    <time>
     <hour>8</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QLabel" name="lhasta_sabado">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Hasta</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeFin_sabado">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>22</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2000</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="maximumTime">
    <time>
     <hour>22</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
   <property name="minimumTime">
    <time>
     <hour>8</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>20</y>
     <width>389</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lnombre">
        <property name="styleSheet">
         <string notr="true">&lt;B&gt;&lt;/B&gt;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Nombre:     &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineNombre">
        <property name="statusTip">
         <string comment="Ejemplo" extracomment="Ejemplo"/>
        </property>
        <property name="inputMask">
         <string comment="Ejemplo" extracomment="Ejemplo"/>
        </property>
        <property name="text">
         <string comment="Ejemplo" extracomment="Ejemplo"/>
        </property>
        <property name="placeholderText">
         <string comment="Ejemplo" extracomment="Ejemplo"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Limitar franja&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="limitarFranja">
      <property name="text">
       <string>Si</string>
      </property>
      <property name="iconSize">
       <size>
        <width>26</width>
        <height>26</height>
       </size>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>195</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QPushButton" name="ok">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="cancelar">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineNombre</tabstop>
  <tabstop>timeInicio</tabstop>
  <tabstop>timeFin</tabstop>
  <tabstop>boxAulas</tabstop>
  <tabstop>boxDIa</tabstop>
  <tabstop>ok</tabstop>
  <tabstop>boxAnio</tabstop>
  <tabstop>cancelar</tabstop>
  <tabstop>cargaHoraria</tabstop>
 </tabstops>
 <resources>
  <include location="imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
